cli/lib/main.js

```diff
@@ -9,12 +9,17 @@ const minimist = require('minimist');
 const { load } = require('./config');
 
 async function main(argv) {
-  if (!argv[0]) {
-    argv[0] = 'help';
-  }
+  argv[0] = argv[0] || 'help';
```

Comment:
Is mutating argv safe, or would it make sense to do something like `const args = [...argv]; args[0] = args[0] || 'help'` here?

Reply:
Can you expand on what you mean by safety?  What are some dangerous side effects you expect could occur?

This was the effect of the original code from `master`.

Reply:
My main concern is about mutating actual `process.argv`, which could cause side effects if, for example, someone is executing `ds` from another script. But I just read the rest of the file and saw `main(process.argv.slice(2))`. Sorry I missed that before!

Reply:
Great, so for posterity the `argv` we're dealing with is a different object, so you won't get those types of unexpected side effects.